test/e2e/e2e_setup_test.go

```diff
@@ -302,14 +303,16 @@ func addVCSimTestVariables(managementClusterProxy framework.ClusterProxy, specNa
 	}
 
 	err = managementClusterProxy.GetClient().Create(ctx, envVar)
-	Expect(err).ToNot(HaveOccurred(), "Failed to create EnvVar")
+	if !apierrors.IsAlreadyExists(err) {
+		Expect(err).ToNot(HaveOccurred(), "Failed to create EnvVar")
+	}
 
 	Eventually(func() bool {
 		if err := managementClusterProxy.GetClient().Get(ctx, crclient.ObjectKeyFromObject(envVar), envVar); err != nil {
 			return false
 		}
 		return len(envVar.Status.Variables) > 0
-	}, 30*time.Second, 5*time.Second).Should(BeTrue(), "Failed to get EnvVar %s", klog.KObj(envVar))
+	}, 300*time.Second, 5*time.Second).Should(BeTrue(), "Failed to get EnvVar %s", klog.KObj(envVar))
```

Comment:
This seems like a very long time. Same below

Is this really that slow?

Reply:
TODO(chrischdi): lower and run test.

Note: took this over from fabrizio's work.

Reply:
Tested with 100 clusters / 20 concurrency and old values seem to be good.

Reply:
Took up to `43` seconds in my test at max. so let's go to 60

Reply:
Thx for testing!